### ISSUE TITLE
Invalid Plugin

### ISSUE THREAD

**Issue by mboneil10** [umts/pvta-multiplatform#390]
Set up multiplatform on my computer, but it took more steps than those in the readme. There was an invalid plugin error after running `ionic serve -c`. I added a new plugin, which made `ionic serve -c` run as expected. Not sure what caused this problem or if the fix was appropriate.

**Comment by akaplo** [comment]
Interesting. Can you provide the exact error/failure output and the commands you used to fix it?

**Comment by mboneil10** [comment]
I checked out to master to recreate the error, but that branch doesn't produce it. So, I can't provide the exact error. Maybe I made a mistake earlier on, so here's what happened. 

I setup multiplatform. There was one error popping up after `ionic serve -c:`
```
[ERROR] Error occurred while loading plugins. CLI functionality may be limited.
        Checking for CLI updates now...
[ERROR] No updates found after plugin error--please report this issue.
``` 
(Based off of my search history from last night)

At this point, I looked back at the readme... I hadn't read the "Testing on Devices" section. This is totally my fault. So, I updated the software of my computer. Don't know if that should have fixed anything. However, when I ran `ionic serve -c`, the result was a different error message:

 `[ERROR] PLUGIN INVALID`. 

Can't find the exact message; that's what I copied and pasted into Google. The change I made in the branch I'm working on is below, along with the commands:
```
npm install --save-dev --save-exact @ionic/cli-plugin-ionic-angular@latest
npm install --save-dev --save-exact @ionic/cli-plugin-cordova@latest
```
The diff of `package.json`:
```
-    "@ionic/cli-plugin-cordova": "0.0.9",
-    "@ionic/cli-plugin-ionic-angular": "1.0.0",
+    "@ionic/cli-plugin-cordova": "1.4.0",
+    "@ionic/cli-plugin-ionic-angular": "1.3.1",

``` 
So... I updated the plugins and made them angular-specific? I have a feeling this is my fault, but just in case it isn't, I put the issue up here.

**Comment by akaplo** [comment]
I was able to reproduce that same unhelpful error message on a clean copy of master, so this isn't your fault at all!  Updating the plugin versions seems to solve the problem, so I assume we were trying to use old plugin versions on a new version of the CLI and it didn't like that.

I've created a [cli branch](https://github.com/umts/pvta-multiplatform/tree/cli) and opened a [pr](https://github.com/umts/pvta-multiplatform/pull/391) that simply updates the plugin versions.
